src/components/grid-aware/Footer/InfoBlock/InfoBlock.module.css

```
@@ -0,0 +1,44 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.infoBlock {
+  background-color: var(--color-midnight-blue);
+  color: var(--color-gray-300);
+  display: grid;
+  font-family: var(--font-family-lato);
+  font-size: 14px;
+  font-weight: 300;
+  grid-column: 1 / span 12;
+  line-height: 16.8px;
```

You generally want to use [unitless line height in CSS](https://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/); otherwise, you may have strange behavior for child elements, since they'll inherit the line height in absolute terms. Here, the unitless value would be 1.2.

```suggestion
  line-height: 1.2;
```

---

ok